packages/cashier/src/components/page-container/page-container.tsx

```diff
@@ -23,7 +23,7 @@ const PageContainer: React.FC<React.PropsWithChildren<TProps>> = observer(
                 {!is_loading && (
                     <div className='page-container__content'>
                         {!is_mobile && left && <div className='page-container__sidebar--left'>{left}</div>}
-                        <ThemedScrollbars className='page-container__main'>
+                        <ThemedScrollbars className='page-container__main' height='calc(100svh - 8rem)'>
```

Comment:
For the reference 👇 , i substract it with `8rem` because page header and navbar use 8rem of the view height.

![image](https://github.com/binary-com/deriv-app/assets/125247833/63dbf656-d87e-4a6c-bdbb-8005af61a7cc)